package.json

```diff
@@ -8,9 +8,9 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "puppeteer": "16.1.0",
-    "youtube-dl-exec": "2.1.5",
-    "dotenv": "16.0.1"
+    "dotenv": "^16.0.3",
```

Comment:
I don't recommend using `^` for specifying versions in `package.json`.

My reasoning is that if we explicitly control the versions, we always know what versions of the dependencies are being used, and if we run into some issue that is not caused by our code, we can try to update our dependency versions so we fix the issue.

Please let me know if there is a good reason for using `^` that I am missing.